MapleServer2/Constants/Skills/JobSkillOrder.cs

```diff
@@ -0,0 +1,94 @@
+using System.Collections.Generic;
+
+namespace MapleServer2.Constants.Skills {
+    public static class JobSkillOrder {
+        public static Dictionary<int, int[]> Order = new Dictionary<int, int[]>();
+
+        static JobSkillOrder() {
+            Order.Add(10, new int[] {
+                10100091, 10100261, 10100041, 10100211, 10100161, 10100281, 20000011, 10100061, 10100011, 10100181,
+                10100062, 10100131, 10100081, 10100031, 10100201, 10100151, 10100101, 20000001, 10100051, 10100221,
+                10100001, 10100171, 10100291, 10100071, 10100021, 10100191, 10100141, 10100294, 10100111, 10100162,
+                10100231, 10100251, 10100202, 10100271, 10100204, 10100121, 10100241
```

Comment:
Any way we can move these into either a db that handles these constants or a file?

Reply:
I'm going to second this. We should really create db files for this type of thing rather than baking it into the code.

Reply:
Could move them into an xml or something but don't really see the point of doing that just to parse it and end up with the same objects.

Reply:
The reason for moving it out is mainly for organization, we would handle similar functions in a similar manner. Better organization improves maintainability, even if it provides no performance gains.